Clean styles after PF4 migration

aljesusg opened the pull request:
Fix https://github.com/kiali/kiali/issues/1950

After remove patternfly and some bootstrap stuff that we don't need by PF4 some styles must be fixed

## Build status 

### Before
```bash
  1.56 MB        build/static/js/2.83dd604d.chunk.js
  154.64 KB      build/static/js/main.65cb81fa.chunk.js
  76.48 KB       build/static/css/main.29ca52ea.chunk.css
  63.24 KB       build/static/css/2.1f669268.chunk.css
  735 B          build/static/js/runtime-main.7ce183d8.js
```

### Now
```bash
  1.39 MB        build/static/js/2.d4e36797.chunk.js
  165.21 KB      build/static/js/main.a392a3ee.chunk.js
  66.01 KB       build/static/css/2.ba43579b.chunk.css
  3.84 KB        build/static/css/main.18faf7f6.chunk.css
  735 B          build/static/js/runtime-main.7ce183d8.js
```
So we reduce the size

aljesusg commented:
mmm I didn't know that the graph panel use the bootstrap styles of panel-heading so I added the styles.

jshaughn commented:
> mmm I didn't know that the graph panel use the bootstrap styles of panel-heading so I added the styles.

@aljesusg If there is a fairly easy alternative it's fine by me if we make a change.

aljesusg commented:
> > mmm I didn't know that the graph panel use the bootstrap styles of panel-heading so I added the styles.
> 
> @aljesusg If there is a fairly easy alternative it's fine by me if we make a change.

Yep I did the change, I added the styles that you are using from bootstrap, the result is the same @jshaughn , anyway we'll need to review it . Thanks ;)

hhovsepy commented:
Just an early testing,
alignment of header, breadcrumb and tabs:
![Screenshot from 2020-03-03 14-16-32](https://user-images.githubusercontent.com/604313/75779406-d5489e00-5d59-11ea-9fa5-34a83b1dcb9a.png)

aljesusg commented:
> Just an early testing,
> alignment of header, breadcrumb and tabs:
> ![Screenshot from 2020-03-03 14-16-32](https://user-images.githubusercontent.com/604313/75779406-d5489e00-5d59-11ea-9fa5-34a83b1dcb9a.png)

Thanks @hhovsepy I am working in this style now.

@lucasponce you can now check the slider
@jshaughn these change could impact the graph replay, I checked it and it seems that is ok.

Let the hunger games begin...I mean the review games ^^
Thanks !!!

mtho11 commented on src/pages/ServiceDetails/ServiceInfo/__tests__/__snapshots__/ServiceInfoDescription.test.tsx.snap:
```diff
@@ -23,15 +23,25 @@ exports[`#ServiceInfoDescription render correctly with data should render servic
+        <Stack
+          gutter="md"
+          style={
+            Object {
```
This `Object { ` seems a bit strange to me. Is this needed?

aljesusg replied:
Mmm the point is that this was updated by yarn test -u ...let me check why

xeviknal replied:
This object seems pretty normal to me, in the snapshot context. The style field needs an object to be defined. Though, it is not necessary in the Component itself. 
I've also gone through the other snapshots an I see the ones having the style field also has the object next to it.

mtho11 replied:
I don't think it is needed but since it is generated we won't worry about it.

aljesusg commented:
> LGTM in the context that this is a big PR and I think it's better to consolidate.
> 
> We can't merge with CI problems, I think we need to find a fix/workaround for that.
> 
> There are some minor tweaks about style:
> 
> ![image](https://user-images.githubusercontent.com/1662329/75977444-b9213a00-5edc-11ea-8843-f28fedcab2c4.png)

This is in master too
> 
> ![image](https://user-images.githubusercontent.com/1662329/75977453-bf171b00-5edc-11ea-9ec1-abb11e90cb63.png)
> 

> ![image](https://user-images.githubusercontent.com/1662329/75977463-c4746580-5edc-11ea-8273-17eebeb5f0e9.png)
> 
> ![image](https://user-images.githubusercontent.com/1662329/75977480-c8a08300-5edc-11ea-971f-184f008fbf31.png)
> 
> ![image](https://user-images.githubusercontent.com/1662329/75977492-cf2efa80-5edc-11ea-9ca4-3796c9ef4f74.png)
> 
> I'm fine to address those in a separate task, but this PR can't be merged until CI is green.

Anyway let me fix this, they are minors one

aljesusg commented:
@lponce I did the changes but I don't know what is wrong in the image related with weights

About the image related with the bell and question icon. Did you mean here the alignment of icons? In that case this is related with patternfly and this is happening in master too. Maybe we should ask to UX team about this.

aljesusg commented:
Can someone check if yarn test works in local? 
I updated my local node like the travis version and I don't know why travis fail.

xeviknal commented:
7196de46 commit. `yarn test` works well.
